site/components/Navigation/index.tsx

```diff
@@ -279,50 +263,48 @@ export const Navigation: FC<Props> = ({
             />
             <NavItemMobile
               name={t({ message: "App", id: "shared.app" })}
-              toggledNavItemID={toggledNavItemID}
-              setToggledNavItemID={setToggledNavItemID}
               id="App"
```

Comment:
You can now remove all `id` from every `NavItemMobile` here
remove the prop too
and delete the type `NavItemMobile`

Reply:
removed